client/js/models/archive.js

```diff
@@ -3,7 +3,12 @@ module.exports = {
   namespace: 'archive',
   state: {
     key: null,
-    file: null
+    file: null,
+    numPeers: 0,
+    signalhubs: [
+      'signalhub.mafintosh.com',
+      'signalhub.dat.land'
```

Comment:
Ok, sorry for back and forth. There is detailed discussion about this in #163. TL;dr let's server-render state directly into server-side template for now and then upgrade later to xhr.